packages/core/execution-context/core-execution-context-browser-internal/src/execution_context_service.ts

```diff
@@ -98,6 +98,7 @@ export class ExecutionContextService
 
   private getDefaultContext() {
     return {
+      type: 'application',
```

Comment:
The default context should have a `type` to ensure there's something in it. Otherwise, the `x-opaque-id` can look like `kibana::appId`.